conference_101719.tex

```diff
@@ -0,0 +1,195 @@
+Thus, using the network should be avoided whether it is on the same path or not because the risk of being attacked is extremely high. 
+In contrast, in the case where a RAP clones the MAC address which the LAP far from the user device has or it does not clone, we cannnot detect only by checking the accordance of MAC addresses.
+Thus the further investigation of the existence of another AP on the path is needed.
+
+\subsubsection{Setting MAC Address on user device}
+In this procedure, in order to reveal the existence of LAP which is connected on the other side of the targeted AP if  the user device is attacked, the situation is intentionally created where the MAC address of LAP is duplicated by the user device.
+Thus, the MAC address of user device is set to $M_{Ai}$ in preparation for the next procedure.
+
+\subsubsection{ARP Observation}
+After the MAC address is set to $M_i$, the connection between the user device and $A_{target}$ is once lost since the gateway become unable to use original MAC address or $M_{i-1}$ as a destination for the packets.
+Therefore, the gateway needs to upgrade the ARP table, in which the sets of MAC addresses and IP addresses of all the devices in the network are registered, so as to continue the communication with it.
+Consequently, the ARP works automatically with new $M_i$ set to user device.
+Thus, in this procedure, these ARP packets are observed so as to investigate whether the $A_i$ is on the same path with the user device. ここってso as toの主語とメインの主語が異なるけど問題ないのかなぁ．
+
+%ARP is a procedure always done at the beginning of the Wi-Fi communication for mapping IP address to MAC address in a LAN to establish communication to the Internet and composed of two phases: ARP request, and ARP reply.
+%ARP request is a packet sent by a user device to a gateway i.e. router of the LAN at the beginning of the network communication.
+%Since MAC address is used for a destination address of a packet in LAN communication, the device cannnot communicate with the Internet, or even gateway, without the MAC address of the gateway.
+%In order to acquire a MAC address of a gateway, ARP request packets, which includes source MAC address information, are sent to the gateway's IP address. 
+%After the gateway get the packet from the device, ARP reply packets are sent by the gateway to the source device on the basis of the source MAC address included in ARP request packets, which can tell the gateway's MAC address by including it as a source MAC address.
+As a result, in non attack scenario or an attack scenario of $A_i\ne LAP$, which means $A_i$ is on distinct path, ARP reply packets from the gateway are observed by the user device within the detection time, represented as $t$.
+In contrast, in the case of $M_{A_i}=M_{LAP}$ in an attack scenario, ARP reply packets from the gateway cannot be observed since the LAP on the way back to the source of ARP request, i.e. the user device, receive them.
```

Comment:
```suggestion
In contrast, in the case of $M_{A_i}=M_{LAP}$ in an attack scenario, ARP reply packets from the gateway cannot be observed since the LAP on the way back to the source of ARP request, i.e. the user device, receive them.
```